lib/sendf.c

```diff
@@ -311,7 +349,32 @@ ssize_t Curl_recv_plain(struct connectdata *conn, int num, char *buf,
                         size_t len, CURLcode *code)
 {
   curl_socket_t sockfd = conn->sock[num];
-  ssize_t nread = sread(sockfd, buf, len);
+  ssize_t nread;
+#ifdef _WIN32
```

Comment:
This piece of logic should also be moved into a separate function, only to be called on Windows.